package.json

```diff
@@ -1,7 +1,7 @@
 {
   "description": "Official TinyMCE Angular Component",
-  "author": "Ephox Inc",
-  "license": "Apache-2.0",
+  "author": "Tiny Technologies",
```

Comment:
Apparently the correct value of "author" is ["Ephox Corporation DBA Tiny Technologies, Inc."](https://github.com/tinymce/tinymce/blob/develop/modules/tinymce/package.json#L10) (or so I was told by Lee).

Note that I still have "Tiny Technologies" on the tinymce-react repo but I have been putting it on the new stuff.